include/openssl/dsa.h

```diff
@@ -199,7 +199,7 @@ int (*DSA_meth_get_sign_setup(const DSA_METHOD *dsam))
 int DSA_meth_set_sign_setup(DSA_METHOD *dsam,
         int (*sign_setup) (DSA *, BN_CTX *, BIGNUM **, BIGNUM **));
 int (*DSA_meth_get_verify(const DSA_METHOD *dsam))
-        (const unsigned char *, int , DSA_SIG *, DSA *);
+        (const unsigned char *, int, DSA_SIG *, DSA *);
```

Comment:
should we have a DSA_meth_verify_func_t or similar typedef?  Makes the code simpler.  would also need a doc update.  but also it's okay if not for this PR.